design-toolkit/components/package.json

```diff
@@ -52,6 +62,7 @@
     "bench": "pnpm vitest bench --run --dir src",
     "build": "pnpm generate && pnpm license && pnpm format && tsup && pnpm build:css",
     "build:css": "pnpm tailwindcss -i ./src/index.css -o ./dist/styles.css",
+    "constellation-tracker": "node ../../tooling/constellation-tracker/dist/main.js",
```

Comment:
Shouldn't there be a devDependency on this package and then it should be executed from node_modules bin?